medusa/clients/nzb/nzbget.py

```diff
@@ -127,7 +127,7 @@ def sendNZB(nzb, proper=False):
     nzbcontent64 = None
     if nzb.result_type == 'nzbdata':
         data = nzb.extra_info[0]
-        nzbcontent64 = standard_b64encode(data)
+        nzbcontent64 = standard_b64encode(data).decode()
```

Comment:
I don't really understand why this is needed. What error are you getting?

Reply:
i got 
```
2019-04-27 18:24:12 INFO     SNATCHQUEUE-SNATCH-121361 :: [f46bfac] Sending NZB to NZBget
2019-04-27 18:24:12 WARNING  SNATCHQUEUE-SNATCH-121361 :: [f46bfac] Connect Error to NZBget: could not add ... to the queue
```
and after a bit of debugging I got it to spit out
```
2019-04-27 19:45:43 DEBUG    SNATCHQUEUE-SNATCH-121361 :: [f46bfac] Exception:<Fault 2: 'Invalid parameter (NZBContent)'>
```
I then looked at standard_b64encode and saw that it changed from python 2.7 to 3
$ cat test.py
from base64 import standard_b64encode
data = b'long string with lots of actual data'
nzbcontent64 = standard_b64encode(data)
print(type(nzbcontent64))

$ python2.7 test.py
<type 'str'>

$ python3 test.py
<class 'bytes'>
```

Reply:
Good catch! 👍 

Can you also add the `decode()` to this line? https://github.com/pymedusa/Medusa/blob/b40a1cb7d700d03eda9032bce1b4bdb6f51bd1ac/medusa/clients/nzb/nzbget.py#L160
It would cause the same error otherwise. After that I think we can merge the PR.

Reply:
Nvm, I added it myself.

Reply:
sorry i haven't touched this computer since Sunday, i'll do a quick regression test with python 2.7 and nzbget to make sure it still works

Reply:
test passed successfully from develop branch with latest nzbget